app/assets/javascripts/map/services/SentinelService.js

```diff
@@ -1,18 +1,18 @@
-define([
-  'Class',
-  'uri',
-  'bluebird',
-  'map/services/DataService'
-], function(Class, UriTemplate, Promise, ds) {
-
+/* eslint-disable */
```

Comment:
Do we need this service for the sentinel tiles inside the backbone app? Is this due to the react implementation?